### PR TITLE
[MI-2948] Fixed the issue that link and unlink commands are available for all users

### PR DESCRIPTION
**Pull request by manojmalik20**
Created two utils for checking if a user is channel admin or system admin
Added the logic for hiding connect and disconnect bot commands for all except system admins

### CODE REVIEW

**Review by ayusht2810 — APPROVED** [review]

**Review by ayusht2810 — APPROVED** [review]

File: server/command.go [hunk 1]
```diff
@@ -125,6 +127,22 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 }
 
 func (p *Plugin) executeLinkCommand(c *plugin.Context, args *model.CommandArgs, parameters []string) (*model.CommandResponse, *model.AppError) {
+	isSysAdmin, err := p.IsSystemAdmin(args.UserId)
+	if err != nil {
+		return p.cmdError(args.UserId, args.ChannelId, "Something went wrong.")
```
Comment (avas27JTG) [inline server/command.go:132]: should not we pass the actual error here?

Comment (Kshitij-Katiyar) [reply server/command.go:132]: If not at least log the error

File: server/command.go [hunk 2]
```diff
@@ -179,6 +197,22 @@ func (p *Plugin) executeLinkCommand(c *plugin.Context, args *model.CommandArgs,
 }
 
 func (p *Plugin) executeUnlinkCommand(c *plugin.Context, args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
+	isSysAdmin, err := p.IsSystemAdmin(args.UserId)
+	if err != nil {
+		return p.cmdError(args.UserId, args.ChannelId, "Something went wrong.")
+	}
+
+	if !isSysAdmin {
+		isChannelAdmin, err := p.IsChannelAdmin(args.UserId, args.ChannelId)
```
Comment (Kshitij-Katiyar) [inline server/command.go:206]: i think we can fix lint errors coming from our code. Like here we have shadow declaration for err

File: server/command.go (hunk 1 above)
Comment (manojmalik20) [reply server/command.go:132]: No, because the user doesn't need to know the actual error.

**Pull request merged**